repctl/pkg/cmd/templates/powerstore_source.yaml

```diff
@@ -16,6 +16,7 @@ parameters:
   {{- end }}
   {{ .ReplicationPrefix }}/remoteSystem: {{ .Parameters.RemoteSystem.Source }}
   {{ .ReplicationPrefix }}/rpo: {{ .Parameters.Rpo }}
+  {{ .ReplicationPrefix }}/Mode: {{ .Parameters.Mode }}
```

Comment:
Same as the previous comment for '/Mode'

Reply:
taken care